lib/astutils.cpp

```diff
@@ -1212,45 +1212,46 @@ SmallVector<ReferenceToken> followAllReferences(const Token* tok,
             return x.token < y.token;
         }
     };
-    SmallVector<ReferenceToken> refs_result;
     if (!tok)
-        return refs_result;
+        return {};
     if (depth < 0) {
-        refs_result.push_back({tok, std::move(errors)});
+        SmallVector<ReferenceToken> refs_result;
+        refs_result.emplace_back(ReferenceToken{tok, std::move(errors)});
```

Comment:
sorry .. doesn't it work like this?

    refs_result.emplace_back(tok, std::move(errors));

just want to double check that the normal syntax can't be used?